src/validators/employment.js

```diff
@@ -164,7 +164,7 @@ export class EmploymentValidator {
   validReference() {
     return (
       new NameValidator(this.referenceName).isValid() &&
-      validPhoneNumber(this.referencePhone) &&
+      validPhoneNumber(this.referencePhone, { numberType: false }) &&
```

Comment:
@apburnes @hursey013 Is this something we need to do all over the application, since most don't have a number Type? Or should this be done in the inverse manner (i.e., explicitly turn it on for the 1? 2? places where number type is used)

Reply:
I like that better. What should the default validation be? With `numberType` or without `numberType`?

Reply:
Without, there is only one spot in the entire application that uses `numberType` (when applicant enters their own contact phone numbers).